### DISCUSSION TITLE
Generate Project in x86

### DISCUSSION THREAD

**Discussion by Ernaw** [NVIDIA-Omniverse/PhysX#254] [Q&A]
Hello, after 5.X it's possible to generate the project in x86  Windows ? i try to modify the template but without success

**Comment by preist-nvidia** [comment]
Hi @Ernaw - we no longer provide 32bit build configs, so you would have to set this up yourself. You can also check the community-maintained (but suspected inactive) build config fork: https://github.com/NVIDIA-Omniverse/PhysX?tab=readme-ov-file#community-maintained-build-configuration-fork
